Submitted By: Joe Gifford <jim@linuxfromscratch.org>
Date: 2006-09-05
Upstream Status: Unknown
Origin: Jim Gifford
Description: Change /lib/udev to /lib64/udev

diff -Naur udev-098.orig/Makefile udev-098/Makefile
--- udev-098.orig/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/Makefile	2006-09-05 11:09:28.984721185 -0700
@@ -96,7 +96,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev
 udevdir =	/dev
diff -Naur udev-098.orig/extras/ata_id/Makefile udev-098/extras/ata_id/Makefile
--- udev-098.orig/extras/ata_id/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/ata_id/Makefile	2006-09-05 11:09:28.984721185 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/cdrom_id/Makefile udev-098/extras/cdrom_id/Makefile
--- udev-098.orig/extras/cdrom_id/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/cdrom_id/Makefile	2006-09-05 11:09:28.988721081 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/dasd_id/Makefile udev-098/extras/dasd_id/Makefile
--- udev-098.orig/extras/dasd_id/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/dasd_id/Makefile	2006-09-05 11:09:28.988721081 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/edd_id/Makefile udev-098/extras/edd_id/Makefile
--- udev-098.orig/extras/edd_id/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/edd_id/Makefile	2006-09-05 11:09:28.992720977 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/firmware/Makefile udev-098/extras/firmware/Makefile
--- udev-098.orig/extras/firmware/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/firmware/Makefile	2006-09-05 11:09:28.992720977 -0700
@@ -13,7 +13,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/floppy/Makefile udev-098/extras/floppy/Makefile
--- udev-098.orig/extras/floppy/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/floppy/Makefile	2006-09-05 11:09:28.996720872 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/path_id/Makefile udev-098/extras/path_id/Makefile
--- udev-098.orig/extras/path_id/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/path_id/Makefile	2006-09-05 11:09:28.996720872 -0700
@@ -13,7 +13,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/run_directory/Makefile udev-098/extras/run_directory/Makefile
--- udev-098.orig/extras/run_directory/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/run_directory/Makefile	2006-09-05 11:09:29.000720768 -0700
@@ -15,7 +15,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/scsi_id/Makefile udev-098/extras/scsi_id/Makefile
--- udev-098.orig/extras/scsi_id/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/scsi_id/Makefile	2006-09-05 11:09:29.004720664 -0700
@@ -19,7 +19,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/usb_id/Makefile udev-098/extras/usb_id/Makefile
--- udev-098.orig/extras/usb_id/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/usb_id/Makefile	2006-09-05 11:09:29.004720664 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/volume_id/Makefile udev-098/extras/volume_id/Makefile
--- udev-098.orig/extras/volume_id/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/volume_id/Makefile	2006-09-05 11:09:29.008720559 -0700
@@ -14,7 +14,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-098.orig/extras/volume_id/lib/Makefile udev-098/extras/volume_id/lib/Makefile
--- udev-098.orig/extras/volume_id/lib/Makefile	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/extras/volume_id/lib/Makefile	2006-09-05 11:09:29.008720559 -0700
@@ -5,8 +5,8 @@
 # Released under the GNU General Public License, version 2.
 #
 includedir = 	${prefix}/usr/include
-libdir =	${prefix}/lib
-usrlibdir =	${prefix}/usr/lib
+libdir =	${prefix}/lib64
+usrlibdir =	${prefix}/usr/lib64
 
 INSTALL = /usr/bin/install -c
 INSTALL_DATA  = ${INSTALL} -m 644
diff -Naur udev-098.orig/udev_rules_parse.c udev-098/udev_rules_parse.c
--- udev-098.orig/udev_rules_parse.c	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/udev_rules_parse.c	2006-09-05 11:12:24.532142387 -0700
@@ -432,9 +432,9 @@
 				if (pos)
 					pos[0] = '\0';
 
-				/* allow programs in /lib/udev called without the path */
+				/* allow programs in /lib64/udev called without the path */
 				if (strchr(file, '/') == NULL) {
-					strlcpy(file, "/lib/udev/", sizeof(file));
+					strlcpy(file, "/lib64/udev/", sizeof(file));
 					strlcat(file, value, sizeof(file));
 					pos = strchr(file, ' ');
 					if (pos)
diff -Naur udev-098.orig/udev_utils_run.c udev-098/udev_utils_run.c
--- udev-098.orig/udev_utils_run.c	2006-08-22 16:32:25.000000000 -0700
+++ udev-098/udev_utils_run.c	2006-09-05 11:09:32.512629158 -0700
@@ -124,9 +124,9 @@
 		}
 	}
 
-	/* allow programs in /lib/udev called without the path */
+	/* allow programs in /lib64/udev called without the path */
 	if (strchr(argv[0], '/') == NULL) {
-		strlcpy(program, "/lib/udev/", sizeof(program));
+		strlcpy(program, "/lib64/udev/", sizeof(program));
 		strlcat(program, argv[0], sizeof(program));
 		argv[0] = program;
 	}
